Assets/Dialogue/Barks/BrutusOfficeBarks/BrutusOfficeIdleBark 3.asset

```diff
@@ -0,0 +1,15 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 3a4c60ef165185e4d9c0e38f4bee7913, type: 3}
+  m_Name: BrutusOfficeIdleBark 3
+  m_EditorClassIdentifier: 
+  line: Who dis?     Brutus    Hehehe
```

Comment:
I'm confused by this line LOL like why he's saying this in idle LOL (sounds like maybe at the most he would say this in the mirror or something)
also the "dis" seems too slangy, maybe we should keep it to regular speech / old-timey speech for the residents?

Reply:
loooooooool couldnt come up with anything else but I'll change it